docs/docs/types/gate.md

```diff
@@ -0,0 +1,46 @@
+A more invasive approach to data collection or user identification. This type of module prevents users from viewing content or navigating until some sort of data has been collected and processed. Highly effective when promoting white papers or other online materials that are not to be freely available.
+
+``` javascript
+var module = pathfora.SiteGate({
+	// module settings
+});
+
+pathfora.initializeWidgets([ module ]);
+```
+
+### SiteGate - [Live Preview](../../examples/preview/types/gate/gate.html)
```

Comment:
This is getting added to the menu somehow:

![image](https://cloud.githubusercontent.com/assets/174740/15916693/39311df4-2daa-11e6-9389-ed6dcfce7030.png)